yt/yt/server/lib/exec_node/config.cpp

```diff
@@ -146,6 +146,9 @@ void TSlotManagerConfig::Register(TRegistrar registrar)
 {
     registrar.Parameter("locations", &TThis::Locations);
 
+    registrar.Parameter("enable_disk_quota", &TThis::EnableDiskQuota)
```

Comment:
Maybe quotas?

Reply:
In volume-manager config it's called "enable_disk_quota"

Reply:
I've redesigned it into per slot-location option.